android/app/build.gradle

```diff
@@ -81,6 +82,9 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "com.google.firebase:firebase-analytics-ktx"
+    implementation "com.google.firebase:firebase-crashlytics-ktx"
+    implementation platform('com.google.firebase:firebase-bom:31.0.0')
```

Comment:
if u use flutter Crashlytics plug and configure it by flutter fire configure, u do not need to add android dependencies, it already has in plugin.
check here 
https://firebase.google.com/docs/crashlytics/get-started?platform=flutter